frontend/src/modules/EditZing/Components/EmailModal.tsx

```diff
@@ -56,6 +134,76 @@ export const EmailModal = ({
     )
   }
 
+  /** Step 2
+   *  Show email send failure screen
+   *  Try Again Button */
+  const StepFailure = () => {
+    return (
+      <Box
+        sx={{
+          display: 'flex',
+          justifyContent: 'center',
+          alignItems: 'center',
+          paddingTop: '8%',
+        }}
+      >
+        <Typography variant="h5" component="h5" fontWeight={'400'}>
+          Uh oh... Something went wrong when trying to send the email. Try again
+          to reauthenticate. <br></br>
+          <br></br>
+          <em>
+            {' '}
+            *This will automatically attempt to send the email one more time.{' '}
+          </em>
+        </Typography>
+        <Button
+          onClick={() => {
+            adminSignIn().then(() => handleEmailSend())
+          }}
+          sx={{ position: 'absolute', bottom: '20%' }}
+        >
+          Try Again{' '}
```

Comment:
Are the random space characters to be inserted in the HTML in this component and the others needed

Reply:
Yeah, I don't think they are. Just my editor settings... which I should probably turn off.